saltax/contexts.py

```diff
@@ -55,23 +57,22 @@
 XNT_COMMON_CONFIG = straxen.contexts.xnt_common_config.copy()
 
 # fuse based saltax options overrides
-FXNT_COMMON_OPTS_REGISTER = XNT_COMMON_OPTS["register"].copy()
-FXNT_COMMON_OPTS_REGISTER.remove(straxen.Peaklets)
-FXNT_COMMON_OPTS_REGISTER.remove(straxen.PulseProcessing)
-FXNT_COMMON_OPTS_REGISTER = [
+SXNT_COMMON_OPTS_REGISTER = XNT_COMMON_OPTS["register"].copy()
```

Comment:
Actually, these about `SXNT_*` should happen in https://github.com/XENONnT/saltax/pull/161